### PR TITLE
Create functionality

### PR DESCRIPTION
**Pull request by Drewbrad4**
Added our form to our lend page, with its handleChange and handleSubmit. Our handleSubmit currently runs a function called createNewBook that simply logs the form entry. Styled a bit just for visuals.

### CODE REVIEW

**Comment by devJoelMichael** [comment]
Looks good!

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -2,9 +2,16 @@
...
-    height: 100vh;
+    margin-bottom: 10vh;
 }
 
+.title {
+    margin: 1vh 2vw 1vh 2vw;
+}
+
+
+// .................................................. Header
```
Comment (talk2MeGooseman) [inline app/assets/stylesheets/application.scss:13]: You should try making different scss files to logically chunk your styles for easier management. Then they can be imported into your application.scss file

**Review by devJoelMichael — APPROVED** [review]
Looks good

File: app/javascript/components/App.js [hunk 2]
```diff
@@ -20,27 +20,33 @@ import NotFound from './Pages/NotFound'
...
-  render () {
+  render() {
     const {
       logged_in,
       sign_in_route,
       sign_up_route,
       sign_out_route,
       current_user
     } = this.props
+    console.log(current_user);
```
Comment (talk2MeGooseman) [inline app/javascript/components/App.js:45]: Logger still lurking

**Pull request merged**